sources/background_subtractor/bgsubstractor.cpp

```diff
@@ -121,7 +120,7 @@ namespace tmd {
         mask_copy.release();
         checked_pixels.release();
         m_frame_index += m_step_size;
-        step();
+        m_input_video.set(CV_CAP_PROP_POS_FRAMES, m_frame_index);
```

Comment:
HUGE speed up when removed for steps of one frame.
Need to mesure it and do it only when its worth.